### ISSUE TITLE
Ascending or descending order of timeline posts?

### ISSUE THREAD

**Issue by basbiezemans** [CodeForEindhoven/IdeaGarden#24]
Posts are currently ordered from old to new (top to bottom). At some point it will take a lot of scrolling to reach the latest post. I suggest a reverse order, which will show the latest post at the top.

**Comment by milovanderlinden** [comment]
Currently there is NO order. We need timestamps to order. Will check the
impact.

**Comment by goodnekk** [comment]
That's not true, there is an order. You can simple call .reverse() on the array.
